jupyter_notebook/notebookapp.py

```diff
@@ -261,6 +263,22 @@ def init_handlers(self, settings):
         new_handlers.append((r'(.*)', Template404))
         return new_handlers
 
+    def _create_widget_shim_handlers(self, settings):
+        """Creates a handler that shims the loading of widgets
+        JS files for the old /static/widgets/js/*.js path."""
+        class DeprecationHandler(IPythonHandler):
+            def get(self, url_path):
+                self.set_header("Content-Type", 'text/javascript')
+                self.finish("""
+                    console.warn('`/static/widgets/js` is deprecated.  Use `/nbextensions/widgets/widgets/js` instead.');
+                    define(['%s'], function(x) { console.log('...loaded...'); return x; });
+                """ % url_path_join(settings['base_url'], 'nbextensions', 'widgets', 'widgets', url_path))
```

Comment:
I have a feeling it's the last bullet here :http://requirejs.org/docs/errors.html#timeout